waspc/dev-tool.project

```diff
@@ -8,4 +8,5 @@ jobs: $ncpus
 constraints:
   ormolu ==0.4.0.0,
   stan ==0.0.1.0,
-  hlint ==3.3.6
\ No newline at end of file
+  hlint ==3.3.6,
+  prune-juice ==0.7
```

Comment:
well not so much because prune-juice has some kind of error :D, but I filed an issue so hopefully they will fix it! It did report couple of redundant deps though, but those were all the ones we want to keep for some special reason. So it seems to mostly work, but not completely, and I hope it will once they fix it. I still left it as it seemed useful!